org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -182,9 +182,9 @@ public SpaceMode DistanceSpaceMode
         private float WorldToLocalScale => transform.InverseTransformVector(transform.forward).magnitude;
 
         /// <summary>
-        /// Holds the duples Collider + Interactor that have triggered proximity.
+        /// Holds the interactors that trigger proximity.
         /// </summary>
-        private HashSet<(Collider, XRBaseInteractor)> activeCollidersWithInteractor = new HashSet<(Collider, XRBaseInteractor)>();
+        private HashSet<NearInteractionModeDetector> activeInteractors = new HashSet<NearInteractionModeDetector>();
```

Comment:
Rename `activeInteractors` to `activeDectectors`

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/673633711a758de353c35b7f90ab544115cf627b, resolving.